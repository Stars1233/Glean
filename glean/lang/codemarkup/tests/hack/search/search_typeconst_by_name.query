query: hack.SearchTypeConstByName { name = "T" }
transform: [normord, []]
perf: true
