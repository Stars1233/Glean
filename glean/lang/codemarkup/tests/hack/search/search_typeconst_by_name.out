[
  "@generated",
  {
    "key": {
      "decl": {
        "typeConst": {
          "key": {
            "container": {
              "class_": {
                "key": {
                  "name": {
                    "key": {
                      "name": { "key": "SourceClass" },
                      "namespace_": { "key": { "name": { "key": "NS" } } }
                    }
                  }
                }
              }
            },
            "name": { "key": "T" }
          }
        }
      },
      "name": "T",
      "parent": {
        "key": {
          "name": { "key": "SourceClass" },
          "namespace_": { "key": { "name": { "key": "NS" } } }
        }
      }
    }
  }
]